<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scalingMethod</class>
 <widget class="QWidget" name="scalingMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>970</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>вернуться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImageButton">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>850</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImageButton">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>910</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="test_image">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тестовое</string>
   </property>
  </widget>
  <widget class="QLabel" name="scaling_answer">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ</string>
   </property>
  </widget>
  <widget class="QLabel" name="test_scaled">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тестовое скейл</string>
   </property>
  </widget>
  <widget class="QLabel" name="scaling_answer_scaled">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ скейл</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовое изображение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ скейлинга</string>
   </property>
  </widget>
  <widget class="QWidget" name="histogramWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>1231</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="histogramWidget" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>1231</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовое отмасштабированное</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ отмасштабированный</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
